packages/web-app-pdf-viewer/src/App.vue

```diff
@@ -57,6 +55,8 @@ export default {
         this.loading = true
         const response = await this.getFileContents(fileContext.path, { responseType: 'blob' })
         this.blobUrl = URL.createObjectURL(response.body)
+        const fileInfo = await this.getFileInfo(this.currentFileContext.path, {})
```

Comment:
```suggestion
        const fileInfo = await this.getFileInfo(fileContext.path, {})
```
fileContext is already in the params of `loadPdf`